js/packages/teams-ai/src/Application.ts

```diff
@@ -570,6 +576,27 @@ export class Application<TState extends TurnState = TurnState> {
                     return false;
                 }
 
+                // Populate {{$temp.input}}
+                if (typeof state.temp.input != 'string') {
+                    // Use the received activity text
+                    state.temp.input = context.activity.text;
+                }
+
+                // Download any input files
+                if (Array.isArray(this._options.fileDownloaders) && this._options.fileDownloaders.length > 0) {
```

Comment:
Do we want to streamline the way users could have downloaded images put to storage?

Reply:
Defer to later.

Reply:
I'm persisting them to `state.temp` because they don't need to live beyond the current turn